### PR TITLE
Resolves #173

### LINKED ISSUE #173: React Components
How can traditional react components be used from htm?

[React-widgets](https://jquense.github.io/react-widgets/) seem to work off-the-shelve with preact-compat and jsx, but not with htm...

```
import { html, Component } from 'htm/preact';
import style from './style';
import Multiselect from 'react-widgets/lib/Multiselect';
import 'react-widgets/dist/css/react-widgets.css';
      
export default class Profile extends Component {
        render({}, {}) {
//return <div class={style.profile}><Multiselect data={['alphabet','alfalfa','broccoli']} /></div>; //jsx works
return html`<div class=${style.profile}><Multiselect data=${['alphabet','alfalfa','broccoli']} /></div>`; //htm doesn't work yet...
        }
}
```

Or, how would a javascript noob go about debugging this? In chrome's Elements view, the htm version has this showing up as simply \<multiselect data="alphabet,alfalfa,broccoli"></multiselect>

### PR DESCRIPTION
**Pull request by spoerri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,9 +28,9 @@ It uses standard JavaScript [Tagged Templates] and works in [all modern browsers
 
 The syntax you write when using HTM is as close as possible to JSX:
 
-- Spread props: `<div ...${props}>`
+- Spread props: `<div ...${props}>` intead of `<div ...{props}>`
```
Comment (bebraw) [inline README.md:31]: /s/intead/instead

Comment (developit) [reply README.md:31]:
```suggestion
- Spread props: `<div ...${props}>` instead of `<div ...{props}>`
```

**Review by developit — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -28,9 +28,9 @@ It uses standard JavaScript [Tagged Templates] and works in [all modern browsers
 
 The syntax you write when using HTM is as close as possible to JSX:
 
-- Spread props: `<div ...${props}>`
+- Spread props: `<div ...${props}>` instead of `<div ...{props}>`
```
Comment (developit) [inline README.md:31]:
```suggestion
- Spread props: `<div ...${props}>` instead of `<div {...props}>`
```

**Comment by developit** [comment]
Thanks @spoerri!

**Pull request merged**